### ISSUE TITLE
Recommendations/Similar Movies

### ISSUE THREAD

**Issue by MrConqueror** [recloudstream/cloudstream#351]
### Describe your suggested feature

It's better to Place them in the bottom. Or "More like this" text like Netflix app. Instead that ugly notebook icon above 

### Other details

![IMG_20230201_194041](https://user-images.githubusercontent.com/102837820/216032960-23960cc9-8123-476e-a1a2-6efa9ade9a8a.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by MrConqueror** [comment]
> swipe

I meant those movies recommendations should be place in the bottom.

**Comment by LagradOst** [comment]
I understand that, but I mean that the button is just an accessibility button and you can swipe instead

**Comment by MrConqueror** [comment]
> I understand that, but I mean that the button is just an accessibility button and you can swipe instead

Yeah ik we can just swipe right I actually never click that notebook, but what I was referring is those movies in recommendations should be in the bottom because that's where it should be and it looks fine.

**Comment by oathzed** [comment]
I disagree. How it currently is is best. If you remove the side swipe menu, where exactly are those posters going to go for tv shows? If you have tv shows how it currently is and movies on the bottom, that's bad UX.
